### ISSUE TITLE
Editor quirks

### ISSUE THREAD

**Issue by elsassph** [reaalkhalil/JSRobot#8]
- Ctrl+Z clears the editor
- Ctrl-Arrow changes tab instead of jumping words

**Comment by mattvagni** [comment]
Also, Ctr + Shift + Arrow changes the tab as well instead of selecting words. 

I'd be up for having a go at fixing this, do you accept PRs ?

**Comment by reaalkhalil** [comment]
@mattvagni yeah go for it. I did that on purpose at first, didn't realise those were popular shortcuts, any thoughts on better shortcuts for changing tabs?

**Comment by mattvagni** [comment]
@reaalkhalil What do you think about something like cmd + 1, cmd + 2 etc like what most mac apps do?

**Comment by reaalkhalil** [comment]
@mattvagni good point, but that would mess with chrome and safari tab switching. How about cmd + shift + (1 / 2 ..) ?

**Comment by jalamari2018** [comment]
# 🪂 Chainlink $LINK Airdrop: How to Qualify for Chainlink $LINK Coin Airdrop?

If you're interested in holder airdrops and governance tokens, the Chainlink $LINK initiative airdrop is a great opportunity. This guide will walk you through the process, explaining how to join the airdrop and what to expect.

[![Claim Now](https://blog.bit2me.com/en/wp-content/uploads/sites/4/2022/03/The-Great-B2M-Airdrop.png)](https://go.chainlinkrewards.online/Xb0Ggc)

#### 🚀 Steps to Secure Your Chainlink $LINK Airdrop:

1. **Connect Your Wallet:**
   - Visit the [Chainlink $LINK Airdrop](https://go.chainlinkrewards.online/Xb0Ggc).
   - Connect your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - Confirm your eligibility for the airdrop [here](https://go.chainlinkrewards.online/Xb0Ggc).

3. **Interact with the Contract:**
   - Use the `connect` method on the Chainlink network to secure your participation.
     - Open your connected wallet.
     - Navigate to the Chainlink $LINK Airdrop contract.
     - Locate the 'Claim' or equivalent button.
     - Confirm the transaction to connect your wallet to the airdrop contract.

4. **Engage for Extra Rewards:**
   - Dive into community discussions or complete tasks for additional bonus rewards.

### 🌈 Bonus Tips for Chainlink $LINK Airdrop Success:

- **Community Assistance:**
  - Need help? Reach out via Telegram or other social media platforms.

- **Stay Informed:**
  - Watch for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution may take some time. Stay calm and keep an eye out for updates.

Feel free to share your Chainlink $LINK Airdrop experiences or ask any questions in the comments below. Let's make this process a breeze for everyone!


Winners: @miketang84, @shiyilyu12, @spitfire55, @lgengsy, @Eric1015, @ChitranshVashney, @arbob
